request-manager/grapher/spec.go

```diff
@@ -93,6 +94,9 @@ func ReadConfig(configFile string) (Config, error) {
 		sequence.Name = sequenceName
 		for nodeName, node := range sequence.Nodes {
 			node.Name = nodeName
+			if node.Parallel != nil && *node.Parallel == 0 {
+				return cfg, fmt.Errorf("Parallel: 0 in sequence %s, node %s", sequence.Name, node.Name)
```

Comment:
The best error messages tell you how to fix the problem they're reporting. So in this case, I would add `Parallel must be > 0` or something similar at the end of the message.

Reply:
ah also, it's good to think about what your error will look like if a higher-level caller wraps it, eg `got error from X: <insert your error here>`. To that end it's usually a good idea to start with a lowercase letter (unless you're referring to a proper noun that's always capital).